packages/docs/src/routes/docs/components/context/index.mdx

```diff
@@ -97,7 +97,7 @@ Let's dig into every API involved:
 
 ### `createContextId()`
 
-This method is used to create a new `Context`.
+This method is used to create a new `Context ID`.
```

Comment:
Again, sorry for not making it clear  - 

I'd use etiher `ContextId` (as code)

or "Context ID" as text (wrapped with normal quotation marks)

Reply:
i think it would be better to have "Context ID" in bold WDYT?